src/front/feedback/FeedbackContainer.tsx

```diff
@@ -105,9 +110,7 @@ extends React.Component <any, FeedbackContainerState> {
 			alert(<FormattedMessage id="fb.errors.emptyErr"/>);
 			return;
 		}
-
 		console.log('sending feedback');
```

Comment:
Forgot to remove console.log?

Reply:
Need to clean code little bit ) and remove debug comments

Reply:
legacy, removed